src/ucs/datastruct/ptr_array.c

```diff
@@ -54,16 +49,37 @@ ucs_ptr_array_freelist_set_next(ucs_ptr_array_elem_t *elem, unsigned next)
                     (((ucs_ptr_array_elem_t)next) << UCS_PTR_ARRAY_NEXT_SHIFT);
 }
 
+static inline unsigned
+ucs_ptr_array_freelist_get_size_free_ahead(ucs_ptr_array_elem_t elem)
```

Comment:
this func is identical with `ucs_ptr_array_size_free_get`.
Please remove this func.

Reply:
Agreed, removed.